driver_test.go

```diff
@@ -515,9 +515,9 @@ func invalidUserPassErrorTests(invalidDNS string, expectedErr int, t *testing.T)
 
 func TestBogusHostNameParameters(t *testing.T) {
 	invalidDNS := fmt.Sprintf("%s:%s@%s", username, pass, "INVALID_HOST:1234")
-	invalidHostErrorTests(invalidDNS, []string{"no such host", "verify account name is correct", "HTTP Status: 403", "Temporary failure in name resolution", "server misbehaving"}, t)
+	invalidHostErrorTests(invalidDNS, []string{"no such host", "verify account name is correct", "HTTP Status: 403", "Temporary failure in name resolution", "server misbehaving", "connection broken"}, t)
 	invalidDNS = fmt.Sprintf("%s:%s@%s", username, pass, "INVALID_HOST")
-	invalidHostErrorTests(invalidDNS, []string{"read: connection reset by peer", "EOF", "verify account name is correct", "HTTP Status: 403", "Temporary failure in name resolution", "server misbehaving", "failed to auth"}, t)
+	invalidHostErrorTests(invalidDNS, []string{"read: connection reset by peer", "EOF", "verify account name is correct", "HTTP Status: 403", "Temporary failure in name resolution", "server misbehaving", "failed to auth", "connection broken"}, t)
```

Comment:
by the way is there a reason why `HTTP Status: 513. Hanging?` is in `invalidHostErrorTest` L#544 and not here?

Reply:
Probably different errors are (or were?) if you connect to correct port and incorrect port (one may be served by GS and another by some proxy etc). But I believe all this messages we keep mostly for historical reasons, for a case like rolling back some parameters etc.